### DISCUSSION TITLE
Can i use it with @vercel/og ?

### DISCUSSION THREAD

**Discussion by nshelia** [vikejs/vike#1128] [Discussion]
I would like to use Vercel's [OG image generation](https://vercel.com/docs/functions/edge-functions/og-image-generation).

When I added /api/og.jsx in my project, it doesn't seem to be in the final build Output folder( Vercel).

am I doing something wrong ?
![Screenshot 2023-09-19 at 5 56 32 PM](https://github.com/brillout/vite-plugin-ssr/assets/27732608/cf10dc27-09c3-4d10-9447-8448d6dbe789)
<img width="754" alt="Screenshot 2023-09-19 at 5 57 40 PM" src="https://github.com/brillout/vite-plugin-ssr/assets/27732608/d8c1f3d7-3309-4c34-a736-c2e5d77d3503">

**Comment by brillout** [comment]
In principle I think so? An example of using it with vite-plugin-ssr would be neat (I'm not aware of any yet).

Keep us updated how it goes.
